tools/sdk-testgen/packages/autorest.testmodeler/webpack.config.js

```diff
@@ -0,0 +1,31 @@
+// @ts-check
+const path = require("path");
+
+/**
+ * @type {import("webpack").Configuration}
+ */
+module.exports = {
+  mode: "production",
+  target: "node",
+  devtool: "source-map",
+  entry: {
+    main: "./src/index.ts",
+  },
+  output: {
+    path: path.resolve(__dirname, "dist", "src"),
+    filename: "[name].js",
+    devtoolModuleFilenameTemplate: "../../[resource-path]",
+  },
+  resolve: {
+    extensions: [".ts", ".tsx", ".js"],
+  },
+  module: {
+    rules: [
+      { test: /\.tsx?$/, loader: "ts-loader" },
+    ],
+  },
+  plugins: [],
+  optimization: {
+    minimize: false,
+  },
+};
```

Comment:
NIT: Missing trailing newline at end of file.

Reply:
yes, thanks! I updated.